inscriptions/ISic030010.xml

```diff
@@ -201,14 +210,17 @@
                     </app>
                 </listApp>
             </div>
-            <div type="translation" xml:lang="en">
-                <p></p>
+            <div type="translation" xml:lang="fr">
+                <p>Je te maintiens dans la crise fatale de la maladie afin que tu ne puisses atteindre tes objectifs.(Dubois)</p>
             </div>
             <div type="translation" xml:lang="it">
                 <p></p>
             </div>
             <div type="commentary" xml:lang="en">
-                <p></p>
+                <p>Il doit s'agir d'une imprécation contre un grabataire dont l'impotence réjouit le rédacteur de la tablette. (Dubois)</p>
```

Comment:
as previous, you can use <p> here and close it at the end of the commentary

Reply:
Sorry, what should be used here? I don't think I see what you mean.

Reply:
you can use one `<p>` element for the entire commentary. you can open `<p>` at the beginning of the commentary and close it at the very end, without entering different `<p></p>` for different lines